### PR TITLE
TC-113 adding pods functions

### PR DESCRIPTION
**Pull request by Matan-Amoyal**
TC-113

### CODE REVIEW

**Comment by Matan-Amoyal** [comment]
@slavik57 please review

File: src/main/groovy/aws/eks/EksPods.groovy [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    // The function return a table of pods in the cluster:
+    // NAME              READY   STATUS             RESTARTS   AGE
+    // hello-log-b92sl   0/1     CrashLoopBackOff   9          24m
+    // hello-log-d5vw8   0/1     CrashLoopBackOff   9          24m
+    // hello-log-hwgwz   0/1     CrashLoopBackOff   9          24m
+    // hello-log-whtw2   0/1     CrashLoopBackOff   9          24m
+    def getPods() {
+        return this.awsCliManager.runAwsCommand(
+            "/bin/bash -c '${this.eksCliManager.getUpdateKubeconfigCommand(this.clusterName)} && kubectl get pods'")
```
Comment (slavik57) [inline src/main/groovy/aws/eks/EksPods.groovy]: You duplicate this code `${this.eksCliManager.getUpdateKubeconfigCommand(this.clusterName)}` every time.
I think you should extract it to some `kubectl` class that has a `runCommand` function.
This function should do: 
```
def runCommand(command) {
  return this.awsCliManager.runAwsCommand(
            "/bin/bash -c '${this.eksCliManager.getUpdateKubeconfigCommand(this.clusterName)} && kubectl {command}'")
}
```
You can do it in a separate pull request

Comment (Matan-Amoyal) [reply src/main/groovy/aws/eks/EksPods.groovy]: Thanks, I will
